charts/posthog/values.yaml

```diff
@@ -869,6 +869,8 @@ zookeeper:
 clickhouse:
   # -- Whether to install clickhouse. If false, `clickhouse.host` must be set
   enabled: true
+  # -- Whether to install `clickhouse` CRDs.
+  installCRDs: true
```

Comment:
Not a huge fan of making this optional as we will likely end up supporting installations with CRDs not in sync with this chart.

Reply:
Not sure I fully understand this point. 
Wouldn't it be like `Posthog version x is compatible with clickhouse version >= y and <=z` 
How is it different from other helm charts that support services with CRDs, like istio?

Reply:
> Wouldn't it be like Posthog version x is compatible with clickhouse chart version >= y and <=z

That yes. I'm more worried about the scenario: 

- customer manually manages the CH operator CRDs
- customer upgrades the Helm chat without upgrading the CRDs if needed
- issues 

This issue is not present if we hard package everything together.